constants.go

```diff
@@ -144,6 +144,13 @@ const (
 
 	// JSON means JSON serialization format
 	JSON = "json"
+
+	// AdminGID is the ID of the standard adm group on linux
+	AdminGID = 4
```

Comment:
What about macOS? Can we lookup the admin GID in a more platform agnostic way?

Reply:
good point, on mac os admin group is not 4, and I'm not even sure it makes sense for MacOS users, so I will limit this logic for linux

Reply:
updated